ext/ffi/ffi.c

```diff
@@ -5367,7 +5373,7 @@ ZEND_MINIT_FUNCTION(ffi)
 	zend_ffi_handlers.has_dimension        = zend_fake_has_dimension;
 	zend_ffi_handlers.unset_dimension      = zend_fake_unset_dimension;
 	zend_ffi_handlers.get_method           = zend_ffi_get_func;
-	zend_ffi_handlers.compare              = NULL;
+	zend_ffi_handlers.compare              = zend_ffi_compare_objects;
```

Comment:
I know that `zend_objects_not_comparable` does not throw an `Error` but it may be better? But this is also fine, just wanted to point out the existence of the "standard" handler.

Reply:
Right, but that doesn't throw. And if we look at some other handlers in FFI where you can compare nonsensical things, they do throw, e.g. https://github.com/php/php-src/blob/9c6282e629742984b88b0b8d5a54d0614d346827/ext/ffi/ffi.c#L1761
So I think to be consistent we should throw here too.

Reply:
Yeah that makes sense.